### PR TITLE
nt-misc/streamlink: bump version & add python 3_6 compat to 9999 ebuild

### PR DESCRIPTION
**Pull request by oczkers**

### CODE REVIEW

File: net-misc/streamlink/streamlink-0.10.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2017 Gentoo Foundation
```
Comment (mgorny) [inline net-misc/streamlink/streamlink-0.10.0.ebuild]: Your calendar seems to be ~3 months off.

**Comment by soredake** [comment]
@mgorny alive, being busy earlier

**Comment by mgorny** [comment]
Just one version, please, and please do not introduce merge commits. Instead, rebase against the latest version of ::gentoo.

**Comment by oczkers** [comment]
Sorry i messed around with unintended push so i'm gonna close this and open new pr https://github.com/gentoo/gentoo/pull/7486

**Pull request closed without merging**